packages/node-client/src/NodeClient.ts

```diff
@@ -165,6 +165,10 @@ class NodeClient implements INodeClient {
   estimateHandlePaymentGas(
     estimateHandlePaymentTxGasDto: EstimateHandlePaymentTxGasDto
   ): Promise<EstimateGasResponse> {
+    const byPass = process.env.BY_PASS_ESTIMATION
+    if ( byPass === 'true' ){
+      estimateHandlePaymentTxGasDto.byPassEstimation = true
```

Comment:
Should check for both string and boolean. Can create a helper method for this in Nodeclient only